extension.ts

```diff
@@ -13,30 +13,32 @@ function lint(textDocument?: vscode.TextDocument) {
   const files = textDocument
     ? [textDocument.fileName]
     : undefined
-  typeCoverage.lint(vscode.workspace.rootPath!, { files, oldProgram, absolutePath: true }).then((result) => {
```

Comment:
`absolutePath: true` is not needed now?

Reply:
TypeScript was throwing an error for absolutePath

Reply:
The CI passed for current code, so not an error.

Reply:
Will add it back